content/docs/getting-started.md

```diff
@@ -77,7 +75,7 @@ We recommend going through [this JavaScript overview](https://developer.mozilla.
 
 If you prefer to **learn by doing,** check out our [practical tutorial](/tutorial/tutorial.html). In this tutorial, we build a tic-tac-toe game in React. You might be tempted to skip it because you're not building games -- but give it a chance. The techniques you'll learn in the tutorial are fundamental to building *any* React apps, and mastering it will give you a much deeper understanding.
 
-### Step-by-Step Guide
+### Conceptual Step-by-Step Guide
```

Comment:
`Step-by-Step` is vague. If it's a conceptual guide, I think it'd be better we explicitly tell people that.